modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/MediaSessionManager.kt

```diff
@@ -89,7 +89,6 @@ class MediaSessionManager(
 
     val mediaSession = MediaSessionCompat(context, "PocketCastsMediaSession")
     val disposables = CompositeDisposable()
-    var seeking = false
```

Comment:
I like that we're able to get rid of this. 👍

Reply:
Yes, it did seem like a bit of a hack.